homeassistant/config.py

```diff
@@ -58,7 +58,7 @@
     LEGACY_CONF_WHITELIST_EXTERNAL_DIRS,
     __version__,
 )
-from .core import DOMAIN as CONF_CORE, ConfigSource, HomeAssistant, callback
+from .core import DOMAIN as DOMAIN_HA, ConfigSource, HomeAssistant, callback
```

Comment:
Let's do `HA_DOMAIN`, that' used 134 times in the core repo, `DOMAIN_HA` is not used a single time.
```suggestion
from .core import DOMAIN as HA_DOMAIN, ConfigSource, HomeAssistant, callback
```